tests/RouteTests.elm

```diff
@@ -188,7 +188,7 @@ fromUrlBasePath =
             \_ ->
                 let
                     url =
-                        mkUrl "/latest/terms/@abc123"
+                        mkUrl "/some-token/ui/latest/terms/@abc123"
```

Comment:
The tests were written that would have caught this, but had a bug in them that mean the base path part was never exercised